src/common/lib/keymaps.c

```
@@ -23,95 +23,223 @@
+ *
+ * @param pos            The position of the key in the keymap (upper nibble = row, lower nibble =
+ * col).
+ * @param make           True if key pressed, false if released.
+ * @param suppress_shift Output parameter set to true if shift should be suppressed for this key.
+ *                       Used by shift-override system when remapping to the final character.
+ *
+ * @return The HID keycode to send, or KC_NO if consumed by layer operation.
  */
-bool keymap_is_action_key_pressed(void) { return action_key_pressed; }
\ No newline at end of file
+uint8_t keymap_get_key_val(uint8_t pos, bool make, bool* suppress_shift) {
```

Cppcheck (2.19.0)

[style] The function 'keymap_get_key_val' is never used.

(unusedFunction)